MusicTaste/MusicTaste/View Controllers/DetailsViewController.m

```diff
@@ -17,10 +18,16 @@ - (void)viewDidLoad {
     //Set Username, Image, and Bio
     self.screenName.text = [self.author valueForKey:@"username"];
     self.bio.text = [self.author valueForKey:@"bio"];
-    NSString *URLString = [self.author valueForKey:@"userimage"];
-    NSString *stringWithoutNormal = [URLString stringByReplacingOccurrencesOfString:@"_normal" withString:@""];
-    NSURL *urlNew = [NSURL URLWithString:stringWithoutNormal];
-    [self.profileView setImageWithURL: urlNew];
+    if ([self.author valueForKey:@"usercustomimage"] != nil){
```

Comment:
If statement to check if a custom image is present and if it is make it the profile picture